metadata.json

```diff
@@ -58,7 +58,7 @@
       "version_requirement": ">= 4.7.0 < 7.0.0"
     }
   ],
-  "pdk-version": "1.10.0",
+  "pdk-version": "1.11.1",
   "template-url": "https://github.com/puppetlabs/pdk-templates#1.10.0",
   "template-ref": "1.10.0-0-gbba9ac3"
```

Comment:
If you were to delete the two line above here and re-run `pdk update` I doubt both would be added back... something seems off about the versions represented in them (not sure though)